mllib-common/src/test/scala/org/apache/spark/ml/attribute/AttributeSuite.scala

```diff
@@ -0,0 +1,242 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.ml.attribute
```

Comment:
Is this file exactly the same with master code file `mllib/src/test/scala/org/apache/spark/ml/attribute/AttributeSuite.scala` ?

Reply:
They are the same. to be safe, I don't touch any test suites in mllib, I just copy them from mllib to mllib-local

Reply:
emm. then we have some code duplication. Can we avoid this ?